### PR TITLE
black: ignore alpha and beta releases.

### PR DESCRIPTION
**Pull request by mhmdanas**
#### Testing the changes
- I tested the changes in this PR: **YES**

[ci skip]

### CODE REVIEW

**Comment by icp1994** [comment]
Maybe it will be better to ignore non-numeric chars all together since there are beta releases too: https://github.com/psf/black/releases

**Comment by mhmdanas** [comment]
Maybe, but back when beta releases for black were used, they were meant to convey the stability of black as a whole, not the stability of that specific release. However, after black was declared stable, beta releases were never used, so we don't know what they'll convey in the future in terms of stability of the release. As a result, I think it's better to be conservative and ignore only alpha releases until a sufficient reason to ignore beta releases too arises.

File: srcpkgs/black/update [hunk 1]
```diff
@@ -0,0 +1 @@
+ignore="*a*"
```
Comment (Piraty) [inline srcpkgs/black/update:1]: this will ignore `1.33.7-all-bugs-squashed`

Comment (mhmdanas) [reply srcpkgs/black/update:1]: Good point, I'll push a fix for it.

Comment (mhmdanas) [reply srcpkgs/black/update:1]: Actually, according to the manual, `ignore` is matched against version numbers, so I think this shouldn't be an issue? Many other packages also do something similar, so if it is an issue, those packages have to be updated too.

Comment (classabbyamp) [reply srcpkgs/black/update:1]: it won't actually ignore those, as it only captures the version part of the regex it uses

Comment (mhmdanas) [reply srcpkgs/black/update:1]: @classabbyamp see my [comment above](https://github.com/void-linux/void-packages/pull/41481#issuecomment-1373753132).

Comment (classabbyamp) [reply srcpkgs/black/update:1]: it's pretty clear what beta would mean in the future, if used again.

Comment (mhmdanas) [reply srcpkgs/black/update:1]: Okay then.

**Pull request merged**